package.json

```diff
@@ -7,6 +7,10 @@
     "@material-ui/core": "^4.11.2",
     "@material-ui/icons": "^4.11.2",
     "@reach/router": "^1.2.1",
+    "@types/jest": "^29.2.3",
+    "@types/node": "^18.11.9",
```

Comment:
out of scope for this PR but should add configuration for specifying which node version to use for the project